### PR TITLE
Implement swagger and pageable

### PR DESCRIPTION
**Pull request by Parashchuk**

### CODE REVIEW

File: src/main/java/bookstore/controller/BookController.java [hunk 1]
```diff
@@ -28,24 +32,28 @@ public class BookController {
 
     @GetMapping
     @ResponseStatus(HttpStatus.OK)
-    public List<BookDto> getAll() {
-        return bookService.findAll();
+    @Operation(description = "Get all books")
+    public List<BookDto> getAll(Pageable pageable) {
+        return bookService.findAll(pageable);
```
Comment (OlehReex) [inline src/main/java/bookstore/controller/BookController.java]: You can add more information to your `@Operation` annotation. For example:
`@Operation(summary = "Get all books", description = "Get list of all books from DB")`

The same to othercases

**Review by OlehReex — APPROVED** [review]
LGTM 😎

File: src/main/java/bookstore/controller/BookController.java [hunk 2]
```diff
@@ -28,24 +32,28 @@ public class BookController {
 
     @GetMapping
     @ResponseStatus(HttpStatus.OK)
-    public List<BookDto> getAll() {
-        return bookService.findAll();
+    @Operation(description = "Get all books")
+    public List<BookDto> getAll(Pageable pageable) {
```
Comment (liliia-ponomarenko) [inline src/main/java/bookstore/controller/BookController.java]: makes sense to add @PageableDefault

File: src/main/java/bookstore/controller/BookController.java (hunk 1 above)
Comment (liliia-ponomarenko) [reply src/main/java/bookstore/controller/BookController.java]: agree

**Review by liliia-ponomarenko — APPROVED** [review]
Good job! But pay attention to comments ;)

**Pull request merged**